.github/workflows/rddepman.yaml

```diff
@@ -0,0 +1,28 @@
+name: Update external dependencies
+on:
+  schedule:
+    - cron: '23 8 * * *'
+  workflow_dispatch: {}
+
+permissions:
+  pull-requests: write
+
+jobs:
+  check-update-versions:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-node@v3
+        with:
+          node-version: '16.x'
+
+      - run: npm ci
+
+      - run: npm run rddepman
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          GITHUB_OWNER: ${{ env.GITHUB_REPOSITORY_OWNER }}
```

Comment:
I'm a little confused by the above diff, but I made it so that the owner name is read from `GITHUB_REPOSITORY_OWNER`, which should allow us to remove the line containing `env.GITHUB_REPOSITORY_OWNER`. I also made it so that the repository name is read from `GITHUB_REPOSITORY`, which has a similar effect.